registry/src/db.rs

```diff
@@ -0,0 +1,12 @@
+// (c) Copyright 2023 Helsing GmbH. All rights reserved.
```

Comment:
should be apache 2 (see the cli)

Reply:
I did not add those comments — they existed in the empty files. I am assuming you want me to copy the license headers from the `cli` and apply them to all of the source files under `registry`.

Reply:
Ah sorry, i did it then 😄

Reply:
Yes!